share/spice/cricket/cricket.js

```diff
@@ -0,0 +1,159 @@
+(function (env) {
+    "use strict";
+    var SCORECARD_ENDPOINT = 'https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20cricket.scorecard%20where%20match_id%3D{$1}&format=json&env=store%3A%2F%2F0TxIGQMQbObzvU4Apia0V0&callback=',
```

Comment:
This should be defined as `alt_to` endpoints in the Perl -- otherwise these are direct client request which expose the user's IP